src/main/java/com/tvtracker/enterprise/dao/MediaEntrySQLDAO.java

```diff
@@ -0,0 +1,43 @@
+package com.tvtracker.enterprise.dao;
+
+import com.tvtracker.enterprise.dto.MediaEntry;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.List;
+
+@Repository("mediaEntryDAO")
+public class MediaEntrySQLDAO implements IMediaEntryDAO {
+
+    @Autowired
+    MediaEntryRepository mediaEntryRepository;
+
+    @Override
+    public boolean save(MediaEntry mediaEntry) throws SQLException, IOException, ClassNotFoundException {
+        mediaEntryRepository.save(mediaEntry);
+        return true;
+    }
+
+    @Override
+    public List<MediaEntry> fetchByUsername(String username) throws SQLException, IOException, ClassNotFoundException {
+        return null;
```

Comment:
This looks like incomplete functionality.
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.